core/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -360,7 +363,7 @@ private TokenParams storedValues() {
          *
          * @return copied object
          */
-        private TokenParams copy() {
+        public TokenParams copy() {
```

Comment:
As I can see it making it public won't matter as you already have access to TokenParams and can use all variables from it.

Reply:
It does matter in a sense that now users will be able to call `tokenParams.copy()`, therefore, we will need to support and maintain this method (removing it would be an API-breaking change). However, this particular method probably will never be removed from the codebase, so IMO it's safe to expose it in the API :wink: